site/content/internal/rd-teams.md

```diff
@@ -51,10 +51,10 @@ weight: 110
 ## Integrations Team
 
 * Lev Brouk - Eng Lead
-* Christopher Speller - Engineer
 * Jason Frerich - Engineer
-* Christopher Poile - Engineer
 * Michael Kochell - Engineer
+* Shota Gvinepadze - Engineer
+* Ben Schumacher - Engineer
```

Comment:
```suggestion
* Ben Schumacher - Engineer
* Daniel Espino Garcia - Engineer
```

Reply:
Oops! I saw Angelos was already here, and forgot to add Daniel. :+1: